### PR TITLE
list command + clean code

### PR DESCRIPTION
**Pull request by Rpsl**
need regenerate phar file

### CODE REVIEW

File: src/Commands/ListGitProfilesCommand.php [hunk 1]
```diff
@@ -23,19 +23,44 @@ class ListGitProfilesCommand extends BaseCommand
...
+                $output->writeln('');
+                $output->writeln('Available profiles:');
+
+                array_walk($profiles, function ($v) use ($output) {
+                    $output->writeln(sprintf('    %s', $v));
+                });
+                exit();
+            }
+
+            $style->error("Profiles not setted.");
```
Comment (AleksandrKuporosov) [inline src/Commands/ListGitProfilesCommand.php]: `Profiles not set.`

Comment (seregayoga) [inline src/Commands/ListGitProfilesCommand.php]: Profiles aren't set

**Comment by AleksandrKuporosov** [comment]
Great request! I think, author must approve it

File: src/Commands/BaseCommand.php [hunk 2]
```diff
@@ -56,14 +59,15 @@ public function switchProfile($profileTitle, $flag = null)
...
+     * Retrieve current git profile
      *
-     * @param  string $flag
-     * @return mixed
+     * @param bool $global
+     *
+     * @return string
      */
-    public function reteriveCurrentProfile($flag = null)
+    public function reteriveCurrentProfile($global = false)
```
Comment (seregayoga) [inline src/Commands/BaseCommand.php]: Rename `reteriveCurrentProfile` to `retrieveCurrentProfile` too

File: src/Commands/ListGitProfilesCommand.php (hunk 1 above)
Comment (AleksandrKuporosov) [inline src/Commands/ListGitProfilesCommand.php]: 🤓

Comment (Rpsl) [inline src/Commands/ListGitProfilesCommand.php]: done

File: tests/Commands/BaseCommandTest.php [hunk 3]
```diff
@@ -29,60 +28,60 @@ public function testMustRunCommandReturnsOutputForValidCommand()
...
+        $this->assertTrue($result);
+    }
+
+    public function testSwitchProfileMustEqualsToReteriveProfileForGlobalFlag()
+    {
+        $flag = 'global';
+        $profileTitle = 'Personal';
+
+        $this->command->switchProfile($profileTitle, $flag);
+        $reterivedProfile = $this->command->retrieveCurrentProfile($flag);
```
Comment (seregayoga) [inline tests/Commands/BaseCommandTest.php]: `reterived` again 🤣

**Comment by zeeshanu** [comment]
@Rpsl Thanks for your PR. 😃 
Could you please look at the use command beacuse it does't seem to be working now.

**Comment by Rpsl** [comment]
@zeeshanu done

**Comment by zeeshanu** [comment]
@Rpsl This command is not working since your first pull request
`git profile use profile_name`

**Comment by Rpsl** [comment]
@zeeshanu sorry, i misunderstood you.
Tell me how you define the error? I use executable ```php ./bin/git-profile``` and all commands work correctly. Maybe it depends on the system? What system do you use?

**Comment by zeeshanu** [comment]
@Rpsl I have windows installed on my system and running commands through cygwin terminal. I have some issues with my public path. After setting my public path correctly use command works fine.

**Comment by Rpsl** [comment]
> After setting my home path correctly use command works fine.

it is perfectly
I understand correctly that more modifications are not required?

**Pull request merged**